src/App.svelte

```diff
@@ -4,15 +4,15 @@
 
   let grossIncome = 0;
   let year = '2022';
-  let marrStatus = 'single';
+  let marrStatus;
   let totalTaxes = 0;
   let afterTax = 0;
 
   function calcTaxes() {
       // calculate standard deduction and taxable income
       let standardDeduction = taxData[year]["standardDeduction"][marrStatus];
       let taxableIncome = Math.max(grossIncome - standardDeduction, 0);
-      
+      console.log(standardDeduction);
```

Comment:
Can you remove unnecessary debug statements?

Reply:
It's Done